cypress/integration/managed_indices_spec.js

```diff
@@ -75,7 +75,7 @@ describe("Managed indices", () => {
     });
   });
 
-  describe("can have policies retried", () => {
+  describe.skip("can have policies retried", () => {
```

Comment:
Why skipping this?

Reply:
I think Ravi's security change breaks this test. Now we don't allow add a non-existent policy to index... We need to come up with new way to write this retry test. Will create an issue for this.

Reply:
https://github.com/opensearch-project/index-management-dashboards-plugin/issues/70